R/igraph-package.R

```diff
@@ -10,6 +9,7 @@
 #' @importFrom rlang inject
 #' @importFrom rlang warn
 #' @importFrom rlang %||%
+#' @import rlang
```

Comment:
needed for the standalone files (unless we prefer I try to find all function names that need to be namespaced in these files) but I didn't want to remove the lines before as I like the clarity... but maybe I should.